covidX/settings.py

```diff
@@ -55,11 +55,12 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = bool(os.getenv("DEBUG_ENV", None))
 
-ALLOWED_HOSTS = [
-    "*",
-    os.getenv("DJANGO_ALLOWED_HOST", "127.0.0.1"),
-    "localhost",
-]
+
+if DEBUG is True:
```

Comment:
You can rewrite the whole thing as:

```python3
ALLOWED_HOSTS = [
    allowed_host if DEBUG and (allowed_host := os.getenv("DJANGO_ALLOWED_HOST")) else '0.0.0.0'
]
```

Make sure to run black afterwards.

Reply:
sure ill do that